packaging/postrm.sh

```
@@ -15,15 +15,53 @@ if [ -f "/lib/systemd/system/$SERVICE_NAME" ]; then
+  fi
+
+  echo "Attempting to remove user $RAWPAIR_USER..."
+  if id -u "$RAWPAIR_USER" &>/dev/null; then
+    userdel "$RAWPAIR_USER" || true
+    echo "User $RAWPAIR_USER deleted."
+  else
+    echo "User $RAWPAIR_USER does not exist."
+  fi
+
+  echo "Attempting to remove group $RAWPAIR_GROUP..."
+  if [ getent group "$RAWPAIR_GROUP" &>/dev/null ]; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)